pkg/teeth/sql/publisher_source.sql

```diff
@@ -0,0 +1,42 @@
+SELECT
```

Comment:
Need a copyright header

Reply:
The sql file is used in a go:embed macro. See https://github.com/abcxyz/github-metrics-aggregator/pull/194/files#diff-643ae6728122b2e55c8522a3413b5969aa2fdbc45383144e50ad12cd32cb30f5R35. Adding the copyright header will embed into the source string. Is this a blocker to using go:embed?

Reply:
All "source" files must include a copyright header.

Reply:
@sethvargo please see the test result if I add the copyright header

```
--- FAIL: TestSetUpPublisherSourceQuery (0.00s)
    publish_logs_test.go:111: embedded source query mismatch  (-want +got):
          (
                """
        +       -- Copyright 2023 The Authors (see AUTHORS file)
        +       --
        +       -- Licensed under the Apache License, Version 2.0 (the "License");
        +       -- you may not use this file except in compliance with the License.
        +       -- You may obtain a copy of the License at
        +       --
        +       --     http://www.apache.org/licenses/LICENSE-2.0
        +       --
        +       -- Unless required by applicable law or agreed to in writing, software
        +       -- distributed under the License is distributed on an "AS IS" BASIS,
        +       -- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
        +       -- See the License for the specific language governing permissions and
        +       -- limitations under the License.
                SELECT
                  pull_request_events.delivery_id,
                ... // 41 identical lines
                """
          )
FAIL
FAIL    github.com/abcxyz/github-metrics-aggregator/pkg/teeth   0.109s
ok      github.com/abcxyz/github-metrics-aggregator/pkg/webhook (cached)
FAIL
```

If this is a blocker, I'll just remove it and use it in the go file like leech does in https://github.com/abcxyz/github-metrics-aggregator/blob/main/pkg/leech/ingest_logs.go#L64.

Reply:
Well yea, you need to update the test to match now 😄 . Add the copyright to the test (or remove any lines that start with `"--"`)

Reply:
Ah d'oh ... I don't know if the https://cloud.google.com/go/bigquery implementation can support comments in the query but we'll find out!